docs/release-guide.md

```diff
@@ -49,3 +51,13 @@ Pull image from the staging registry.
 ```
 docker pull gcr.io/k8s-staging-descheduler/descheduler:v20200206-0.9.0-94-ge2a23f284
 ```
+
+## Helm Chart
+Helm chart releases are managed by a separate set of git tags that are prefixed with `chart-*`. Example git tag name is `chart-v0.18.0`. Released versions of the
```

Comment:
I'd suggest the the tag `chart-0.18.0` would be more idiomatic; `chart-` is functioning like the `v` for a release.

You probably want to describe the chart versioning strategy, starter for 10 below based on the chart aligning with the release `v0.X.Y` version (this could be more flexible with a release `vX.Y.Z` version).

- The major and minor of release and chart must match (e.g. `v0.18.0` => `chart-0.18.0`)
- Patch is not expected to match (e.g. `v0.18.0` => `chart-0.18.n`) but should only be used for patching the chart
- SemVer major or minor changes should only happen after an actual release of the same type

Alternatively you could start the chart at `chart-1.0.0` and use SemVer for the chart; so `v0.18.0` = `chart-1.0.0` and `v0.19.0` = `chart-1.1.0` with `chart-1.1.1` either referring to a chart patch or a patch release on `v0.19.0`.

The second approach would be my suggestion but the first is the way it looks like you're planning to go.

Reply:
Fix in commit bd7fd1d52b433e3a303ec2e1fa2818af65c279ad